lib/internal/modules/cjs/loader.js

```diff
@@ -729,11 +728,8 @@ Module._findPath = function(request, paths, isMain) {
   // For each path
   for (let i = 0; i < paths.length; i++) {
     // Don't search further if path doesn't exist
-    // or doesn't have permission to it
     const curPath = paths[i];
-    if (insidePath && curPath &&
-      ((permission.isEnabled() && !permission.has('fs.read', curPath)) || _stat(curPath) < 1)
-    ) {
+    if (insidePath && curPath && _stat(curPath) < 1) {
```

Comment:
Consider re-introducing permission checks or an alternative security measure to ensure that module loading does not become a vector for security vulnerabilities. [important]